client/css/widgets/holder.css

```diff
@@ -20,6 +20,18 @@
   top: 0;
 }
 
+.holder.showCardBacksToOtherPlayers > .foreign.card.widget {
+  display: block !important;
+}
+
+.holder.showCardBacksToOtherPlayers > .foreign.card.widget > .active.cardFace {
+  display: none;
+}
+
+.holder.showCardBacksToOtherPlayers > .foreign.card.widget > :not(.active).cardFace {
```

Comment:
This is assuming that there is only one other face though right? I've seen a few games where widgets have more than 2 faces where I think this would display all of them.

Reply:
Yes but I don't see how anything in real life with more than 2 faces lets one player see one face and other players 1 specific other face.

Reply:
I mean it doesn't have to be that the extra face has a real life representation. Some games use faces to apply a rotation, which could be used in hanabi for the conventional way that people remember significant cards. But we can always go back and improve that if the need arises.

Reply:
Could add an explicit foreignFace class to the face that other players should see.

Reply:
I did a quick test and only the highest card face will be visible because they are displayed on top of each other. I think this is good enough since probably nobody will ever use this with more than 2 faces.

Reply:
> I did a quick test and only the highest card face will be visible because they are displayed on top of each other. I think this is good enough since probably nobody will ever use this with more than 2 faces.

I did a test on a deck with 4 faces.  For the other players that can see everything, clicking through the faces shows the faces in order as you would expect.  For the player with the `showInactiveFaceToSeat` property, the 4th face was shown three times and the third face was shown once as you click through.

This is not something we need to fix.  Just need to cover it in the wiki that this is only intended for cards with 2 faces.